liquidctl/driver/kraken3.py

```diff
@@ -788,8 +789,15 @@ def check_unsupported_fw_version():
                 self._get_fw_version()
                 if self.fw[0] == 2:
                     raise NotSupportedByDriver(
-                        "setting images is not supported on firmware 2.X.Y, please see issue #631"
+                        "gif images is not supported on firmware 2.X.Y, please see issue #631"
```

Comment:
`is` -> `are` with this sentence.

Also, #631 would be resolved by merging this PR. Is GIF setting not supported at all, or just not implemented right now? Either way, the message should be changed if that issue is to be closed.

Reply:
I did not manage to trigger gif handing in manufacturer software, it sends frames